src/windows/views/timeline_webview.py

```diff
@@ -434,16 +434,17 @@ def ShowTimelineMenu(self, position, layer_id):
         clipboard_tran_ids = [k for k, v in self.copy_transition_clipboard.items() if v.get('id')]
 
         # Paste Menu (if entire clips or transitions are copied)
-        if self.copy_clipboard or self.copy_transition_clipboard:
-            if len(clipboard_clip_ids) + len(clipboard_tran_ids) > 0:
-                menu = QMenu(self)
-                Paste_Clip = menu.addAction(_("Paste"))
-                Paste_Clip.setShortcut(QKeySequence(self.window.getShortcutByName("pasteAll")))
-                Paste_Clip.triggered.connect(
-                    partial(self.Paste_Triggered, MENU_PASTE, float(position), int(layer_id), [], [])
-                )
+        if (self.copy_clipboard or self.copy_transition_clipboard) and (
+            len(clipboard_clip_ids) + len(clipboard_tran_ids) > 0
+        ):
```

Comment:
I kind of want this to be an early return as well, but it hurts my brain too much just trying to work out how the condition should be specified.

Reply:
Simply a `not [what is currently inside the if]` would do it. Should I do it?

Reply:
I would probably add another variable for the current if-condition. Is there a reasonable name?

Reply:
Something like `have_clipboards` should serve well enough, I think. That's really all it's testing: that there are contents (of any sort) on **both** the instance variables for the clipboard lists (self.copy_*_clipboard`) as well as the lists passed in to the function (clipboard_clip_ids`, `clipboard_tran_ids`) — though, no, I can't say that I immediately follow why there would be _**four**_ distinct lists of timeline elements that can be involved in a menu-construction operation, TBH.

Reply:
Ok :-) I've pushed it :-)